Translations/Portuguese/Home.md

```diff
@@ -1,2 +1,9 @@
 This is the link  to the slack bot wiki homepage to be translated.
```

Comment:
Please remove this line :)

Reply:
I have already made the recommended changes